libs/transactions/src/lib/components/transaction-item/transaction-item.component.html

```diff
@@ -9,10 +9,20 @@
   transaction.valueDate | date: 'MMM. d'
 }}</span>
 <span class="transactions-item__description">
-  <span class="transactions-item__merchant">{{
-    transaction.merchant?.name
```

Comment:
Currently there is no merchant visible, since it is not available in the retrieved data.